### PR TITLE
Add checklist setup for PRs

### PR DESCRIPTION
**Pull request by btkostner**
Should fix https://github.com/elementary/appcenter-reviews/issues/37

### CODE REVIEW

File: .github/workflows/review.yml [hunk 1]
```diff
@@ -12,6 +12,24 @@ on:
...
+        with:
+          body: |
+            ## Review Checklist
+
+            - [ ] App opens
+            - [ ] Does what it says
+            - [ ] Matches description in AppData
+            - [ ] Matches screenshot
+            - [ ] AppData custom colors are contrasty
+            - [ ] Name is unique and non-confusing
```
Comment (btkostner) [inline .github/workflows/review.yml:31]: The markdown is configurable, but this is the list that Cass gave so I think it's a pretty good starting point. Easily editable down the road.

**Review by cassidyjames — APPROVED** [review]
Let's do it. Is there a good way to test this, or just merge in and then watch new PRs? :)

File: .github/workflows/review.yml (hunk 1 above)
Comment (cassidyjames) [reply .github/workflows/review.yml:31]: Yeah I was thinking the same. I'm probably missing things but we can iterate as we review things and learn what we need to explicitly check.

**Comment by btkostner** [comment]
Merge and watch :P

I did add tests to this package, so I'm pretty confident it works

**Pull request merged**